bundles/binding/org.openhab.binding.rcswitch/src/main/java/de/pi3g/pi/rcswitch/RCSwitch.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * RCSwitch - port of the Arduino libary for remote control outlet switches.
+ * Arduino library is Copyright (c) 2011 Suat Özgür.  All right reserved.
+ * Ported to Java by Florian Frankenberger 2013.
+ *
+ * Contributors for the Arduino library:
+ * - Andre Koehler / info(at)tomate-online(dot)de
+ * - Gordeev Andrey Vladimirovich / gordeev(at)openpyro(dot)com
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301  USA
+ */
```

Comment:
> You would either have to relicense the while binding to GPL

@teichsta : Please note that this option means that the binding cannot be included to the addons.zip, but would have to be distributed separately, so this isn't really an option.